ansible/playbooks/buildnewportalvms.inventory

```diff
@@ -1,9 +1,10 @@
 [newportalvms]
-# ubuntu22-test is for testing deployment only
-ubuntu22-test.genome.edu.au  ansible_connection=ssh  ansible_user=ubuntu
-
 # this is the development portal - django-portal (in hosts)
 apollo-portal-dev.genome.edu.au  ansible_connection=ssh  ansible_user=ubuntu
 
 # this is the production apollo-portal website
 apollo-portal.genome.edu.au  ansible_connection=ssh  ansible_user=ubuntu  portal_extra_domain=genome.edu.au
+
+[testportalvms]
+# ubuntu22-test is for testing deployment only
```

Comment:
ubuntu22 needs to be a member of newportalvms to access the required vars

Reply:
Ahh of course